### ISSUE TITLE
Broke fontlock when symlink

### ISSUE THREAD

**Issue by osv** [purcell/diredfl#1]
Emacs lock symlink cause broke font lock:
![symlink](https://user-images.githubusercontent.com/213405/31134808-da1a86f0-a86b-11e7-86ad-17ebd818e822.png)

Same but w/o symlink:

![no-symlink](https://user-images.githubusercontent.com/213405/31134811-dd4a22cc-a86b-11e7-8305-ab57904ebe11.png)

**Comment by purcell** [comment]
Thanks for the report. Are you able to paste the text from the first buffer into a comment here, or a gist? That'd make debugging it a bit easier. :-)

**Comment by osv** [comment]
ok :)
```
  /home/olex/work/sync-jira-git/lib:
  total used in directory 32 available 14300716
  drwxrwxr-x 2 olex olex  4096 жов  4 12:52 .
  drwxrwxr-x 7 olex olex  4096 жов  3 18:07 ..
  -rw-rw-r-- 1 olex olex   235 жов  3 18:35 common.js
  -rw-rw-r-- 1 olex olex  3094 жов  3 18:45 prompts.js
  lrwxrwxrwx 1 olex olex    29 жов  4 12:52 .#sync-jira-git.js -> olex@olex-wrk.8149:1507107376
  -rw-rw-r-- 1 olex olex 15260 жов  3 18:48 sync-jira-git.js
```

For testing you can create dir, than touch file `file` and create symlink `ln -s file file2`

**Comment by purcell** [comment]
Great, thanks - fixed!
